pkg/common/file.go

```diff
@@ -20,17 +20,20 @@ import (
 	"io/ioutil"
 	"os"
 	gofilepath "path/filepath"
+	"strconv"
 
-	"gopkg.in/yaml.v2"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
 )
 
 // Config is the internal representation of the yaml that defines
 // server configuration
 type Config struct {
-	RuntimeEndpoint string `yaml:"runtime-endpoint"`
-	ImageEndpoint   string `yaml:"image-endpoint"`
-	Timeout         int    `yaml:"timeout"`
-	Debug           bool   `yaml:"debug"`
+	RuntimeEndpoint string     //`yaml:"runtime-endpoint"`
+	ImageEndpoint   string     //`yaml:"image-endpoint"`
+	Timeout         int        //`yaml:"timeout"`
+	Debug           bool       //`yaml:"debug"`
+	yamlData        *yaml.Node //YAML representation of config
```

Comment:
Can we remove those comments if not needed?
```suggestion
	RuntimeEndpoint string
	ImageEndpoint   string
	Timeout         int
	Debug           bool
	yamlData        *yaml.Node //YAML representation of config
```

Reply:
Sure, removed.